eng/Signing.props

```diff
@@ -1,9 +1,4 @@
 <Project>
-  <ItemGroup>
-    <!-- List of container files that will be opened and checked for files that need to be signed. -->
-    <ItemsToSign Include="$(RepoRoot)bin\packages\$(Configuration)\**\*.nupkg" />
-  </ItemGroup>
-
   <Target Name="ValidateSignFileListIsNotEmpty" BeforeTargets="Sign">
```

Comment:
Should this target be moved to arcade directly?

Reply:
https://github.com/dotnet/arcade/issues/1271

Reply:
Comment below appears wrong:

> Ensure '$(RepoRoot)bin\packages\$(Configuration)' contains nupkg files to sign

Reply:
Yeah I fixed that up in an other commit https://github.com/dotnet/corefx/commit/d77cd27bd6542bc851153f64d9bf9cd02da89bd7.